benchmark/util/promisify.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const common = require('../common');
+const util = require('util');
+const fs = require('fs');
+const zlib = require('zlib');
+const dns = require('dns');
+
+const bench = common.createBenchmark(main, {
+  n: [4e6],
```

Comment:
Not sure what I should have used for n here, I just pulled it from another file. Any suggestions?

Reply:
Anything that has a reasonable low run time with a low flucation?